server/services/interval-cache.js

```diff
@@ -1,7 +1,19 @@
 import IntervalCache from 'interval-cache'; // https://github.com/danneu/interval-cache
 import {getFlags} from '../services/flags-lookup';
+import {getGlobalAlert} from '../services/prismic';
 
 const fiveMinutes = 1000 * 60 * 5;
+
 export const cache = new IntervalCache()
   .every('flags', fiveMinutes, getFlags, [])
+  .every('globalAlert', fiveMinutes, async () => {
+    try {
+      const globalAlert = await getGlobalAlert();
+      return globalAlert;
+    } catch (err) {
+      console.error(err);
+
+      return null;
+    }
```

Comment:
Why is this expected to error?

Reply:
Not expected to, but seems like good hygiene to handle a case where it might? If, for whatever reason, this were to error, we wouldn't want to 500 I don't think.

Reply:
But if it 500s we should know, this will swallow the error silently, which is really bad hygiene.

Reply:
It's a good point about how we bubble errors up to the user.

Reply:
We could start setting up alerting properly, i.e. return null, but notify sentry that something is up?

Reply:
Sounds sensible, yeah